src/item/Item.js

```diff
@@ -216,8 +216,8 @@ new function() { // Injection scope for various item event handlers
         if (flags & /*#=*/ChangeFlag.GEOMETRY) {
             // Clear cached bounds, position and decomposed matrix whenever
             // geometry changes.
-            this._bounds = this._position = this._decomposed =
-                    this._globalMatrix = undefined;
+            this._bounds = this._position = this._decomposed = undefined;
+            Item._clearGlobalMatrixCache(this);
```

Comment:
Could this be an instance method instead, and maybe have a shorter name? 

`this._clearGlobalMatrix();`